scripts/eslint.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This scripts runs ESLint on all relevant JavaScript files in the repository.
+
+set -eu
+
+# Go to the root directory of the repository.
+cd $(dirname $(realpath ${0}))/..
+
+# Find all relevant touched files. Bail out if nothing is found.
+# Note: "d" in --diff-filter means excluding deleted files - otherwise ESLint
+# fails on these non-existing files.
+FILES=$(git diff --name-only --diff-filter=d main -- "*.js")
```

Comment:
Actually let me refactor it into a small helper script (as it's repeated in the `clang-format.sh` script as well).

Reply:
done